danirod/rectball
# Integrate with gdx-gamesvcs

Replaces manual integration with Google Play Services and uses instead gdx-gamesvcs, to offload into a library that I believe will be more tested than Rectball.

Tested using Android Studio in gpeMode that:
* When running in GPE mode, I can:
  * Log in
  * Log out
  * View my achievements
  * View the leaderboard
  * Submit achievements
  * Submit high score

Consequence: the minSdkVersion will have to be bumped to 15. Not a big deal, Android 4.0 => Android 4.0.3. It is still an OS released 10 years ago. I don't expect issues.
